src/kotlin/flwr/src/main/java/dev/flower/android/gRPC.kt

```diff
@@ -0,0 +1,236 @@
+package dev.flower.android
+
+import flwr.proto.FleetGrpc
+import flwr.proto.FleetOuterClass.CreateNodeRequest
+import flwr.proto.FleetOuterClass.CreateNodeResponse
+import flwr.proto.FleetOuterClass.DeleteNodeRequest
+import flwr.proto.FleetOuterClass.DeleteNodeResponse
+import flwr.proto.FleetOuterClass.PullTaskInsRequest
+import flwr.proto.FleetOuterClass.PullTaskInsResponse
+import flwr.proto.FleetOuterClass.PushTaskResRequest
+import io.grpc.ManagedChannel
+import io.grpc.ManagedChannelBuilder
+import io.grpc.stub.StreamObserver
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import java.util.concurrent.CountDownLatch
+import flwr.proto.FlowerServiceGrpc
+import flwr.proto.NodeOuterClass.Node
+import flwr.proto.TaskOuterClass.TaskIns
+import flwr.proto.TaskOuterClass.TaskRes
+import flwr.proto.Transport.ServerMessage
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.coroutineScope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.flow
+
+internal class FlowerGRPC
```

Comment:
We usually `CamelCase` abbreviations like gRPC or HTTP (i.e., `Grpc` / `Http`):
```suggestion
internal class FlowerGrpc
```